"Unsafe code requires the `unsafe' command line option to be specified"

Getting a bunch of errors with the message in the title when importing this package into a new project.

My steps (which I think matches what the docs suggest):

1. I just grabbed the lated Unity HTP (beta 24) and the latest from this repo.  
2. Opened the contents of this repo in Unity and did Assets->Export Package
3. Created a brand new project in Unity and did Assets->Import Package->Custom and chose the thing I just exported

The import completes but the HoloToolkit menu does not appear.  Instead, I see 7 messages in the error list like: "Assets/HoloToolkit/Sharing/Scripts/SDK/AnchorDownloadRequest.cs(58,22): error CS0227: Unsafe code requires the `unsafe' command line option to be specified".  All 7 messages have the same error, just different files.

--- a/GettingStarted.md
+++ b/GettingStarted.md
@@ -14,7 +14,7 @@ https://desktop.github.com/
 
 Open the folder you just cloned in Unity
 
-Now inside of Unity export the package
+Now, inside of Unity ensure you have the Assets folder selected in the project view, and export the package
 
 `Assets -> Export Package…`
 

